src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  text-align: center;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $color---white;
+
+  &__image {
+    width: calc(100% - 6px);
+    height: 134px;
+    background-size: contain;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__title {
+    text-align: start;
+    margin-top: 36px;
+    margin-bottom: 0;
+    color: $color--main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    text-align: start;
+    color: $color--secondary;
+    font-size: 10px;
+    line-height: 14px;
+    margin-top: 4px;
+    margin-bottom: 0;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16px 0 24px;
+
+    .rating__reviews {
+      color: $color--main-accent;
+      font-size: 10px;
+      line-height: 14px;
+      margin: 0;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+
+    .price__name {
+      color: $color--secondary;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    .price__number {
+      color: $color--main-accent;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    width: 100%;
+    height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. just use padding